### PR TITLE
Fix & clean up broken references

### PR DESCRIPTION
**Pull request by cl8n**
notably missing:
- links to sections that don't exist
- links that could be redirects, I didn't check

both of those will be in future PR though.

### CODE REVIEW

File: wiki/Game_Modes/osu!/pt-br.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <!-- wiki -->
-[Game_Modes wikilink]: ../ "Modalides de Jogo"
-[Play_Styles#osu! wikilink]: /wiki/Play_Styles/ "mais informações podem ser encontradas podem ser encontradas sob osu! Estilos de Jogo"
-[Score#osu!SV wikilink]: /wiki/Score/#osu "mais informações podem ser encontradas em osu! Valores de Pontuação"
-[Options#Keyboard wikilink]: /wiki/Options/ "mais informações podem ser encontradas em 'Opções', sob 'Teclado'"
-[Skinning#osu! wikilink]: /wiki/Skinning/osu!/ "osu! Skinning"
-[Mascots#pippi wikilink]: /wiki/Mascots/#pippi "mais informações podem ser encontradas em pippi"
+[Game_Modes wikilink]: .. "Modalides de Jogo"
```
Comment (TPGPL) [inline wiki/Game_Modes/osu!/pt-br.md]: dunno if this works as intended, but i assume its okay

File: wiki/Interface/ko.md [hunk 2]
```diff
@@ -243,7 +243,7 @@
 
 `Mods` 버튼을 누르거나 `F1` 키를 누르면 **[게임 내 모드 선택](/wiki/Game_Modifiers)** 창이 나타납니다.
 
-이 화면에서는 게임 내 모드 (줄여서 `모드`)를 선택할 수 있습니다. 이러한 모드는 게임의 난이도를 낮추지만 얻은 점수의 배율을 낮추기도 하고, 게임의 난이도를 높이지만 얻은 점수의 배율을 높이기도 합니다. 어떤 모드는 완전히 다른 방식으로 게임을 바꾸기도 합니다. [Relax](/wiki/Game_Modifiers/#Relax)와 [Auto Pilot](/wiki/Game_Modifiers/#Auto_Pilot)이 이러한 모드에 속합니다.
+이 화면에서는 게임 내 모드 (줄여서 `모드`)를 선택할 수 있습니다. 이러한 모드는 게임의 난이도를 낮추지만 얻은 점수의 배율을 낮추기도 하고, 게임의 난이도를 높이지만 얻은 점수의 배율을 높이기도 합니다. 어떤 모드는 완전히 다른 방식으로 게임을 바꾸기도 합니다. [Relax](/wiki/Game_Modifiers#Relax)와 [Auto Pilot](/wiki/Game_Modifiers#Auto_Pilot)이 이러한 모드에 속합니다.
```
Comment (TPGPL) [inline wiki/Interface/ko.md]: broken anchor

Comment (cl8n) [reply wiki/Interface/ko.md]: that's one of the things I put in PR description cuz I'll do it later

File: wiki/Game_Modes/osu!/pt-br.md (hunk 1 above)
Comment (cl8n) [reply wiki/Game_Modes/osu!/pt-br.md]: hmm... maybe I should go back and replace all `..` anyway, you pointed out some issue on the website with this a while ago right

Comment (TPGPL) [reply wiki/Game_Modes/osu!/pt-br.md]: i don't remember any `../` issue 🤔 

https://osu.ppy.sh/help/wiki/Game_Modes/osu!?locale=pt-br

first link works fine

File: wiki/osu!stream/Song_List/en.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <!-- wikilink -->
-[osu!stream wikilink]: .,/ "osu!stream"
+[osu!stream wikilink]: /wiki/osu!stream "osu!stream"
```
Comment (TPGPL) [inline wiki/osu!stream/Song_List/en.md]:
```suggestion
[osu!stream wikilink]: .. "osu!stream"
```

Comment (cl8n) [reply wiki/osu!stream/Song_List/en.md]: gonna go the opposite way, it looks like almost nobody uses `..` so I'm just removing all of them. avoids some parsing trouble /shrug

**Pull request merged**